configuration/blockly/rules-blockly-persistence.md

```diff
@@ -34,40 +34,49 @@ More about that topic can be viewed at ![youtube](../images/blockly/youtube-logo
 
 ![statistical-value](../images/blockly/blockly-persistence-get-statistical-value.png)
 
-_Function:_ computes any of the below functions for the given item since the time provided by _ZonedDateTime_-Block
+_Function:_ computes any of the below functions for the given item since the time provided by _ZonedDateTime_-Block, until the time provided by _ZonedDateTime_-Block, or between 2 timepoints provided by _ZonedDateTime_-Blocks.
 
 openHAB supports history and future values.
 A typical example for future values is a weather forecast.
-Due to adding future values in openHAB the amount of attributes has been vastly increased and the names had to be renamed to become more specific in terms of historic and future states.
+
+For each of the functions, you can choose the output type:
+
+- String: output as a string value
+- Quantity: output as a quantity type with unit
+- Number: output as a number
+- Timestamp: time of the persisted value (when applicable)
```

Comment:
Also here (I am bit behind, so it misses one entry=

<img width="181" alt="image" src="https://github.com/user-attachments/assets/a3d5877b-cd49-461c-a73b-4b9b7675315c" />

Reply:
True, although not all functions have it, but I will replace it.

Reply:
Ah OK, thought I had an image in for that, but didn't. The point is: Timestamp is only relevant for some of the analytic actions. E.g., it is not for sum, average, medium..., but it is for getting a persisted state, getting the last update, next update, minimum, maximum... as these results are at a specific point in time. Only when Timestamp is relevant, it is shown in the list. I didn't want to explicitely document this, is the UI should be very much self explaining. And Timestamp will be there when it is relevant. I can put in an image, but not sure it adds much.